dataflow/flex-templates/pipeline_with_dependencies/Dockerfile

```diff
@@ -43,8 +43,8 @@ WORKDIR ${WORKDIR}
 
 COPY requirements.txt .
```

Comment:
I think we can only keep the _toml version and skip the setup.py version, I feel like the difference is insignificant and not specific to Dataflow.